core/database_arango.py

```diff
@@ -1164,6 +1255,11 @@ def filter(
         for name, graph, direction, field in graph_queries:
             graph_query_string += f"\nLET {name} = (FOR v, e in 1..1 {direction} o {graph} RETURN {{ [v.{field}]: e }})"
 
+        acl_query = ""
+        if user and RBAC_ENABLED and not user.admin:
```

Comment:
I wonder if Admin user should always have access to everything. What about an admin account used to create user / enable tasks but that should not access other objects? Just a question, not a strong opinion

Reply:
As discussed, let's do a first iteration like this and then we'll see if we want to tighten things.